manifests/w/Wacom/WacomTabletDriver/6.3.45-1/Wacom.WacomTabletDriver.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Wacom.WacomTabletDriver
+PackageVersion: 6.3.45-1
+PackageLocale: en-US
+Publisher: Wacom Technology, Corp.
+PublisherUrl: https://www.wacom.com/
+PublisherSupportUrl: https://www.wacom.com/en-ca/support
+PrivacyUrl: https://www.wacom.com/privacy
+Author: Wacom Technology, Corp.
+PackageName: Wacom Tablet Driver

```

```suggestion
PackageName: Wacom Tablet
```
to match Control Panel.